citi-onboarding/hacka-pride
# Feature/information

## What I did
* Added the section about information to the page
* Changed `Type.Data` to `Type.Text` on goal model
* Added all the responsivity and requirements to server

## How to test
1. Open the project and run `yarn start` and `nodemon` at the same time
2. Change some informations on the KeystoneJS to verify the usability of admin page
3. Verify all the responsivity to the website
